### ISSUE TITLE
The drop-down lists for "Locales" and "Countries" input fields are not available and no other value then "All" can be entered and accepted

### ISSUE THREAD

**Issue by Softvision-RemusDranca** [mozilla/experimenter#1081]
**[Affected versions]:**
- Firefox Release 66.0 and above
- Chrome 73.0.3683.75

**[Affected Platforms]:** 	 	
- All Windows
- All Mac 	 	
- All Linux

**[Prerequisites]:**
- You are logged with your LDAP account on the "https://experimenter.stage.mozaws.net/" website.
- Have an experiment created on the Experimenter stage website.

**[Steps to reproduce]:**
1. Open the browser with the profile from prerequisites and navigate to "https://experimenter.stage.mozaws.net/".
2. Open the created experiment from prerequisites.
3. Click the "Edit" button from "Population" section.
4. Click the "Locales" input field and observe the behavior.

**[Expected result]:**
- The list with all available locales is displayed and they can be successfully selected.

**[Actual result]:**
- Only the "All locales" option is available and cannot be changed.

**[Notes]:**
- This issue is also reproducible for the "Countries" input field.
- If a locale is manually inserted in the input field, no suggestions are displayed, only the "No result matched "locale" message is displayed. 
- Attached is a screen recording of the issue:
![NoDropdown](https://user-images.githubusercontent.com/40061643/54684210-4c226200-4b1c-11e9-9545-1a010f23878a.gif)

**Comment by peterbe** [comment]
@jaredkerim
This isn't a bug but a matter of having to explicitly ask Ops to execute the `./manage.py load-locales-countries` command on all instances. 
I'll assume you take care of that. 

By the way, once we land the bugzilla settings checking stuff, we can just write another function to that that'd look something like this:

```python
def check_locales_and_countries(app_config, **kwargs):
    if not Locale.objects.all().exists() or not Country.objects.all().exists():
        return [Warning(
            "No locales or countries set up",
            hint="Run './manage.py load-locales-countries'",
            id=EXPERIMENTER_WARNING_LOCALES_AND_COUNTRIES
        )]
    return []
```

**Comment by peterbe** [comment]
Also, I have no idea how the workflow works with OPs for this project but whatever it is, there needs to be a written down post-deploy manual step so this isn't forgotten when they do Prod.

**Comment by Softvision-RemusDranca** [comment]
- I have verified this issue and it's no longer reproducible on the "https://experimenter.stage.mozaws.net/". The locales/countries are displayed in a drop-down list and can be selected. Tested on Windows 10x64, MacOS 10.14 and Ubuntu 18.04 x64.
- However, it seems that all existing experiments are updated with the "Locales" and "Countries" sections with the "All" default option, which could lead to confusion. I opened a new issue expressing this concern, please see issue #1111.
